app-drawer/app-drawer.html

```diff
@@ -204,18 +204,18 @@
         '_updateDocScroll(opened, persistent)'
       ],
 
-      _lastTrackDirection: 0,
-
       _translateOffset: 0,
 
+      _trackEvents: null,
+
+      _startTimeStamp: 0,
+
       attached: function() {
         // Set the scroll direction so you can vertically scroll inside the drawer.
         this.setScrollDirection('y');
 
-        /**
-         * Only transition the drawer shortly after it is attached (e.g. app-drawer-layout
-         * may need to set the initial opened state which should not be transitioned).
-         */
+        // Only transition the drawer shortly after it is attached (e.g. app-drawer-layout
+        // may need to set the initial opened state which should not be transitioned).
         this.$.contentContainer.style.transition = 'none';
         this.async(function() {
```

Comment:
I don't need to wait for a paint - I only need to wait until app-drawer-layout sets its initial opened state. (I updated the comment style because I thought the `/* */` was taking too many lines, and wanted consistent across this file).